### ISSUE TITLE
Outdated elemental-iso-add-registration and related docs

### ISSUE THREAD

**Issue by davidcassany** [rancher/elemental#1039]
The script `elemental-iso-add-registration` is no longer used within the CI and it is currently outdated. We should consider removing it completely in favor of the seed image build procedure. Note this script is also mentioned in several places within docs, as this is used within the image customization story, docs in that area require a revision.

**Comment by kkaempf** [comment]
Agreed, if we keep (and document) a "CLI" way (for scripting/automation etc.)
